Apps/WebClient/src/ClientApp/src/components/shared/pageTitle.vue

```diff
@@ -3,7 +3,7 @@ import Vue from "vue";
 import { Component, Prop } from "vue-property-decorator";
 
 @Component
-export default class PageTitleComponent extends Vue {
+export default class PageTitle extends Vue {
```

Comment:
why are we dropping the "component" word?

Reply:
I discussed this with @FuriousLlama and he thought that \<BaseButtonComponent\> was too wordy and we should also prefer kebab case when using it, so I standardized this across the other shared components I'd created as well.

Reply:
I think there was a miss understanding. I believe we should keep the 'Component' part on the name of the class but not on the aliasing of the components when used

Reply:
Understood.  I can revert that change.